email_validator.gemspec

```diff
@@ -12,15 +12,12 @@ Gem::Specification.new do |s|
     README.md
     CHANGELOG.md
   ]
-  s.files = `git ls-files -- lib/*`.split("\n")
+  s.files = Dir['lib/**/*.rb']
   s.require_paths = %w[lib]
-
-  s.test_files = `git ls-files -- spec/*`.split("\n")
```

Comment:
`test_files` is gone
ref: https://github.com/rubygems/guides/issues/90

Reply:
I don't think that this is correct. Please show me where `test_files` was removed. What commit?